crud/select.lua

```diff
@@ -64,6 +65,8 @@ local function select_on_storage(space_name, index_id, conditions, opts)
         return nil, SelectError:new("Failed to execute select: %s", err)
     end
 
+    tuples = schema.filter_tuples_fields(tuples, opts.field_names)
+
     return tuples
```

Comment:
I think there is no need to assign ``schema.filter_tuples_fields(tuples, opts.field_names)`` ​​in ``tuples``:

```lua
return schema.filter_tuples_fields(tuples, opts.field_names)
```